plugins/poll/lib/poll.rb

```diff
@@ -284,6 +284,10 @@ def self.create!(post_id, poll)
   end
 
   def self.extract(raw, topic_id, user_id = nil)
+    # Poll Post handlers get called very early in the post
+    # creation process. `raw` could be nil here.
+    return [] if raw.nil?
```

Comment:
How about using `.blank?` instead of `.nil?` so it also skip "empty" raws as well?

```suggestion
    return [] if raw.blank?
```